file-formats/dmon/examples/z_aff_example_dmon.dmon.json

```diff
@@ -0,0 +1,10 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "AFF ABAP Daemon Application",
+    "originalLanguage": "en"
+  },
+  "daemonClassName": "CL_AD_EXT_SIMPLE_DAEMON",
+  "callerProgramType": "prog",
```

Comment:
Here I am not sure, whether the values should be upper case?!? It seems this is the R3TR type that shall be entered, isn't it?

Reply:
will be done with #312